flang/lib/Lower/OpenMP.cpp

```diff
@@ -2856,7 +2941,9 @@ genOmpSimpleStandalone(Fortran::lower::AbstractConverter &converter,
                                               currentLocation, opClauseList);
     break;
   case llvm::omp::Directive::OMPD_target_update:
-    TODO(currentLocation, "OMPD_target_update");
+    genEnterExitDataOp<mlir::omp::UpdateDataOp>(converter, semanticsContext,
```

Comment:
Since genEnterExitDataOp() function is now handling Update op as well can you please change this function name to reflect the same? Something like genEnterExitUpdateDataOp() or whichever is appropriate.